modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -721,25 +755,42 @@ More histogram entries ...
 # Thread 8: 45766 56169 56171 ...# 00015 others
 # Thread 9: 02974 08094 13214 ... # 00090 others
 ----
-<1> The docker arguments provided by the user.
+<1> The `podman` arguments provided by the user.
 <2> The user is notified about the measured latency and the configured latency.
 <3> The arguments for the `cyclictest` command.
 <4> The maximum latencies measured on each thread.
 
 [id="cnf-performing-end-to-end-tests-running-oslat"]
 === Running oslat
 
+.Prerequisites
+
+* The Performance Addon Operator provided `PerformanceProfile` is installed and active
+
+.Procedure
+
+. To perform the `oslat`, run the following command: 
+
 [source, terminal]
 ----
-$ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_CPUS=7 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 /usr/bin/test-run.sh -ginkgo.focus="oslat"
+$ podman run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e \
+LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e \
+LATENCY_TEST_CPUS=7 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  \
+registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 /usr/bin/test-run.sh -ginkgo.focus="oslat"
 ----
 
-The above command runs the `cyclictest` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!`.
+The above command runs the `oslat` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!`.
 
 .Example failure output
 [source, terminal]
 ----
-$ docker run -v $KUBECONFIG:/root/kubeconfig:Z -e KUBECONFIG=/root/kubeconfig -e IMAGE_REGISTRY=quay.io/titzhak -e CNF_TESTS_IMAGE=cnf-tests:latest -e PERF_TEST_PROFILE=performance -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=600 -e DISCOVERY_MODE=true -e MAXIMUM_LATENCY=20 -e LATENCY_TEST_CPUS=7 quay.io/titzhak/cnf-tests:latest usr/bin/test-run.sh -ginkgo.v -ginkgo.focus="oslat" <1>
+$ podman run -v $KUBECONFIG:/root/kubeconfig:Z -e KUBECONFIG=/root/kubeconfig -e \
+IMAGE_REGISTRY="my.local.registry:5000/" -e CNF_TESTS_IMAGE=cnf-tests:latest -e \
+PERF_TEST_PROFILE=performance -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_RUN=true -e \
```

Comment:
what's this registry? "my.local.registry:5000/"
AFAIK IMAGE_REGISTRYshould be the official one the customer is using

Reply:
Changed to "registry.redhat.io". Hope that makes more sense